config/config.go

```diff
@@ -35,10 +36,12 @@ import (
 
 //Constant for mesher conf file
 const (
-	ConfFile = "mesher.yaml"
+	ConfFile       = "mesher.yaml"
+	EgressConfFile = "egress.yaml"
```

Comment:
implement a new file handler in archaius to consider egress.yaml as key and file content as value